lib/core/src/components/button/index.ts

```diff
@@ -0,0 +1,16 @@
+import { App as Application } from "vue";
+import Button from "./Button.vue";
+
+import { registerComponent } from '@/utils/plugins';
+
+const Plugin = {
+  install(vue: Application) {
+    registerComponent(vue, Button);
+  }
+};
+
+// Insert Vue use(Plugin);
+
+export default Plugin;
+
+export { Button };
```

Comment:
Hm I'm not really understand this (due to my lack of Vue knowledge). Is there a reference I should read to understand this "Plugin"?

Reply:
@thien-do Please refer this https://v3.vuejs.org/guide/plugins.html

Reply:
In Vue, we can just `install` components once, not need to `import` at every view page